python/rceg3/_RceVersion.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```

Comment:
https://github.com/slaclab/rce-gen3-fw-lib/tree/pre-release/python/RceG3
We did a major overhaul of the rogue software for RCE.  This is old legacy code that you are trying to add back into the pre-release branch. You should delete this obsolete script

Reply:
I did not see the new code anywhere.